programs/mango-v4/src/instructions/token_register_trustless.rs

```diff
@@ -87,7 +87,7 @@ pub fn token_register_trustless(
         net_borrows_in_window: 0,
         borrow_weight_scale_start_quote: 5_000_000_000.0, // $5k
```

Comment:
hmm were these  2 always at this value?

Reply:
yeah - but since borrows are disabled, they don't have an effect